lib/get-last-release.js

```diff
@@ -8,13 +8,15 @@ const getRegistry = require('./get-registry');
 module.exports = async ({publishConfig, name}, logger) => {
   const config = npmConf();
   const tag = (publishConfig || {}).tag || config.get('tag');
-  const {NPM_TOKEN, NPM_USERNAME, NPM_PASSWORD, NPM_EMAIL} = process.env;
+  const {NPM_TOKEN, NPM_USERNAME, NPM_PASSWORD, NPM_EMAIL, LEGACY_TOKEN} = process.env;
   const client = new RegClient(getClientConfig(config));
   const registry = await getRegistry(publishConfig, name);
 
   try {
     const uri = urlResolve(registry, name.replace('/', '%2F'));
-    const auth = NPM_TOKEN ? {token: NPM_TOKEN} : {username: NPM_USERNAME, password: NPM_PASSWORD, email: NPM_EMAIL};
+    const auth = NPM_TOKEN
+      ? {token: NPM_TOKEN}
+      : {username: NPM_USERNAME, password: NPM_PASSWORD, email: NPM_EMAIL, auth: LEGACY_TOKEN};
     auth.alwaysAuth = config.get('always-auth');
```

Comment:
this is too naive, it's needs to check for `always-auth` on both root and scoped.

See https://github.com/npm/npm/blob/0cc9d89ed2d46745f91d746fda9d205fd39d3daa/lib/config/get-credentials-by-uri.js#L24-L29

For now, I just add it in my .npmrc on root as well

Reply:
I tried to figure out how we can end up in a situation with something like that in `.npmrc`:
```
//<registry_url>/:alwaysAuth=true
```
But I couldn't figured out how any `npm` CLI command would result in that.

What's the npm command that write this config?

Reply:
It's returned when authing against artifctory, at least https://www.jfrog.com/confluence/display/RTF/Npm+Registry#NpmRegistry-UsingBasicAuthentication

I couldn't get the npm client to write it, but it does support reading it